pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -993,19 +993,26 @@ func (dm *DockerManager) getSecurityOpt(pod *api.Pod, ctrName string) ([]string,
 		return nil, nil
 	}
 
+	// in Docker 1.10 the operator is :, for anything later it's =
+	optFmt := "%s=%s"
+	if result <= 0 {
```

Comment:
Not sure I am missing something, but above you return when result < 0. Now you are checking result <=0. Do you mean just result = 0 here?